tests/unit/form-specs.tsx

```diff
@@ -5,16 +5,31 @@ import {ReactWrapper, mount} from "enzyme";
 import {Form} from "../../src/components/ContactPage/Form";
 
 describe("<Form/>", () => {
-
     let wrapper: ReactWrapper<any, any>;
+    let node: any;
+
+    const recaptcha = document.createElement("DIV");
```

Comment:
element names should be lowercase

Reply:
Creating elements **MUST** be in `beforeEach`, because some test cases may modify this objects.
It's bad practice when one test case depends on another, because it makes unable to run single test case.